src/plugins/guided_onboarding/server/saved_objects/guided_setup.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { SavedObjectsType } from '@kbn/core/server';
+
+export const guidedSetupSavedObjectsType = 'guided-setup-state';
+export const guidedSetupSavedObjectsId = 'guided-setup-state-id';
+export const guidedSetupDefaultState = {
+  activeGuide: 'unset',
+  activeStep: 'unset',
+};
+export const guidedSetupSavedObjects: SavedObjectsType = {
+  name: guidedSetupSavedObjectsType,
+  hidden: false,
+  // make it available in all spaces for now
+  namespaceType: 'agnostic',
+  mappings: {
+    dynamic: false,
+    properties: {
+      activeGuide: {
```

Comment:
It doesn't seem like you are querying over these fields so there's no need to add a mapping for them. Fields should be thought of like a SQL index, we don't want to index all fields only the ones we're querying/aggregating over.

For more info see:
https://github.com/elastic/kibana/blob/main/dev_docs/tutorials/saved_objects.mdx#L121-L127